doc/topics/proxyminion/index.rst

```diff
@@ -55,10 +127,10 @@ This change was made because proxymodules are a LazyLoader object, but
 LazyLoaders cannot be serialized.  `__opts__` gets serialized, and so things
 like `saltutil.sync_all` and `state.highstate` would throw exceptions.
 
-Also in this release, proxymodules can be stored on the master in
-/srv/salt/_proxy.  A new saltutil function called `sync_proxies` will transfer
-these to remote proxy minions.  Note that you must restart the salt-proxy
-daemon to pick up these changes.
+Support has been added to Salt's loader allowing custom proxymodules
```

Comment:
If this change is relevant to the 2015.8 release, then a new PR for 2015.8 should be made with this change in it.